android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -221,9 +217,7 @@ public void initializeState() {
 
         if (isClearBrowsingDataOnExit()) {
             List<Integer> dataTypes = Arrays.asList(DialogOption.CLEAR_HISTORY,
-                    DialogOption.CLEAR_COOKIES_AND_SITE_DATA, DialogOption.CLEAR_CACHE,
-                    DialogOption.CLEAR_PASSWORDS, DialogOption.CLEAR_FORM_DATA,
-                    DialogOption.CLEAR_SITE_SETTINGS);
+                    DialogOption.CLEAR_COOKIES_AND_SITE_DATA, DialogOption.CLEAR_CACHE);
```

Comment:
I think we're making things fragile here by using DialogOption. If we had a dialog we should be using ClearBrowsingDataFragment, but without one we should be using `BrowsingDataType.HISTORY`, etc... otherwise we have to keep this list in sync `ClearBrowsingDataFragment` and upstream changes could cause the wrong data to be deleted